mlflow/sklearn.py

```diff
@@ -22,6 +22,18 @@
 from mlflow.models import Model
 from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, INTERNAL_ERROR
 import mlflow.tracking
+from mlflow.utils.environment import _mlflow_conda_env
+from mlflow.utils.model_utils import _get_flavor_configuration
+
+FLAVOR_NAME = "sklearn"
+
+DEFAULT_CONDA_ENV = _mlflow_conda_env(
```

Comment:
I would maybe turn this into a function taking a path and returning path instead. Then you don't need to change environemt._mlflow_conda_env and I think global function is preferable to global string

Reply:
Discussed offline, looks good now!